app/assets/stylesheets/users.scss

```diff
@@ -0,0 +1,4 @@
+#promote-search{
+  margin: 0px 10px 0px 5px;
```

Comment:
Why's that? In general, uneven CSS margins/paddings are a flag that there's some other weird problem.